### PR TITLE
mfterm 1.0.7 (new formula)

### PR DESCRIPTION
**Pull request by zhovner**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Adding this to make complete toolset for working with Mifare cards.
Related to [mfoc](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mfoc.rb) and [mfcuk](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mfcuk.rb) that already in brew.

mfterm is packaged in [Kali Linux](http://tools.kali.org/wireless-attacks/mfterm). 

Maybe Dominyk @DomT4 can endorse it.

It has not enough stars for pass audit.

### CODE REVIEW

File: Formula/mfterm.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Mfterm < Formula
+  desc "Terminal for working with Mifare Classic 1-4k Tags"
+  homepage "https://github.com/4ZM/mfterm"
+  url "https://github.com/4ZM/mfterm/releases/download/v1.0.7/mfterm-1.0.7.tar.gz"
+  version "1.0.7"
```
Comment (tdsmith) [inline Formula/mfterm.rb]: You should remove this line.

**Comment by tdsmith** [comment]
I'll wait for @DomT4 to pronounce but one way you could use Homebrew to install this software is to [maintain your own tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md). The linked document has some advice; please let us know if you have any questions and we'll be happy to help you set things up.

File: Formula/mfterm.rb (hunk 1 above)
Comment (zhovner) [inline Formula/mfterm.rb]: Done.

**Comment by zmwangx** [comment]
> GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)

We have to cut a notability line somewhere and I think our current standard is pretty reasonable (50 stars really is very low on GitHub). If Dom would endorse it then sure; otherwise, I don't see how being "related" to mfoc and mfcuk (from nfc-tools) makes it worth inclusion.

Agree with Tim, might be better to maintain it in your own tap.

**Comment by zhovner** [comment]
>I don't see how being "related" to mfoc and mfcuk (from nfc-tools) makes it worth inclusion

Because after you crack keys with  mfoc/mfcuk you need something to manipulate data on the card. I see this as logical continuation.

**Comment by zmwangx** [comment]
> Because after you crack keys with mfoc/mfcuk you need something to manipulate data on the card. I see this as logical continuation.

But anyone can write a "related" utility. If this mfterm is from nfc-tools or otherwise endorsed by them then I would probably consider it a good argument.

**Comment by DomT4** [comment]
I don't have a strong opinion on this, to be honest. It's one of those things that arguments can be made for either way fairly easily. I'm happy to go with any broader team decision here.

It's not a million miles off eligibility, but I'd perhaps lean towards saying this isn't quite in that buffer zone of being close enough to merit dismissing the warning entirely.

Kali is quite a specialist OS, so it doesn't particularly surprise me they'd carry this as it fits their use case perfectly. There's more merit to the packaged elsewhere argument if it's somewhere more generic, like Debian, MacPorts, etc.

**Comment by zhovner** [comment]
>here's more merit to the packaged elsewhere argument if it's somewhere more generic, like Debian, MacPorts, etc.

No any other distro (except Kali Linux) package very specific hack tools like mfoc/mfcuk.
But brew does. So why not complete this toolset?

**Comment by zmwangx** [comment]
Unfortunately they do meet our eligibility requirements, while this one doesn't. We didn't change our code to reject this formula, did we?

**Comment by zhovner** [comment]
Okay, if the only problem is stars quantity, we can wait until it gets enough.
I think one post on ycombinator or slashdot will resolve this.

**Comment by bfontaine** [comment]
It now has 51 stars.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `mfterm 1.0.7 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**